api/capi/src/nntrainer_util.cpp

```diff
@@ -37,13 +37,19 @@ ml_optimizer_to_nntrainer_type(ml_train_optimizer_type_e type) {
 /**
  * @brief Convert nntrainer API layer type to neural network layer type
  */
-nntrainer::LayerType ml_layer_to_nntrainer_type(ml_train_layer_type_e type) {
+const std::string ml_layer_to_nntrainer_type(ml_train_layer_type_e type) {
```

Comment:
as ml_train_layer_type_e and ml::train::LayerType must be kept consistent, it would be good to create a common function which converts int to string and vice versa for both capi and ccapi.

Reply:
Made a function